tracer/src/Datadog.Trace/Propagators/W3CTraceContextPropagator.cs

```diff
@@ -53,7 +53,8 @@ internal class W3CTraceContextPropagator : IContextInjector, IContextExtractor
         {
             new(',', '_'),
             new(';', '_'),
-            new('=', '_'),
+            new('~', '_'),
+            new('=', '~'), // note '=' is encoded as '~' when injecting
```

Comment:
Does this mean the origin could _change_ after propagation?

i.e, the first time, and origin of `synthetics~;,=web` is injected as `dd=o:synthetics___~web`, the next downstream service would be `dd=o:synthetics____web`? or do we never change the `origin` value once it's injected once?